src/cascadia/TerminalSettings/TerminalSettings.cpp

```diff
@@ -20,6 +20,7 @@ namespace winrt::Microsoft::Terminal::Settings::implementation
         _cursorColor{ DEFAULT_CURSOR_COLOR },
         _cursorShape{ CursorStyle::Vintage },
         _cursorHeight{ DEFAULT_CURSOR_HEIGHT },
+        _doubleClickDelimiters{ L" /\\" },

```

Comment:
Why does this default to only three things and the profiles one is a bigger list?

Also, if these should both be the same, they should probably be in a constant shared between both files.